search-app/src/cmr/search/services/humanizers/humanizer_report_service.clj

```diff
@@ -95,3 +105,88 @@
                        "concat humanized rows:" t3))))]
       (info "Create report " t4)
       (str string-writer))))
+
+(defn safe-generate-humanizers-report-csv
```

Comment:
I'm confused now. Isn't this catching errors during report generation? Which is not the same as "the report is empty." I'm inclined to agree with Leo in that this should result in a 5xx status. There should not be a reason why the report could not be generated.